addons/mass_mailing/models/mailing_list.py

```diff
@@ -42,7 +42,7 @@ def _compute_contact_nbr(self):
             ''', (tuple(self.ids), ))
             data = dict(self.env.cr.fetchall())
             for mailing_list in self:
-                mailing_list.contact_nbr = data.get(mailing_list.id, 0)
+                mailing_list.contact_nbr = data.get(mailing_list._origin.id, 0)
```

Comment:
Isn't it linked to the 'if self.ids else 0' global structure of the compute ? Maybe we should use self.ids or [self._origin.id] in the compute.

Reply:
Compute method here follows that structure already. We do the computation only if there are `self.ids`, and yet we get `NewId` in self.